src/compiler/binder.ts

```diff
@@ -174,14 +174,15 @@ namespace ts {
     const binder = createBinder();
 
     export function bindSourceFile(file: SourceFile, options: CompilerOptions) {
-        tracing.begin(tracing.Phase.Bind, "bindSourceFile", { path: file.path });
+        const tracingInfo: tracing.EventData = [tracing.Phase.Bind, "bindSourceFile", { path: file.path }];

```

Comment:
I might have called this "tracingData", given the type.

Reply:
Yeah, that's a typo (I renamed the both a few times...).